### DISCUSSION TITLE
网页标题 增加网站名称后缀

### DISCUSSION THREAD

**Discussion by debuginn** [CaiJimmy/hugo-theme-stack#846] [Ideas]
例如：
```
XXXX文章 - Debug客栈
```
目前生成的页面 html 的 title 标签均没有

**Comment by Lovirr** [comment]
不知道你说的是不是这个意思?
在layouts\partials\head\head.html 中修改以下代码

```html
{{- $title := partialCached "data/title" . .RelPermalink -}}
<title>{{ $title }}</title>
```

```html
{{- $title := partialCached "data/title" . .RelPermalink -}}
<title>{{ $title }} {{.Site.Params.title}}</title>
```

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/106920260/842ff3cc-d299-402c-aebb-c247bcc756fe)

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/106920260/66f1a806-a0d5-4a7d-a938-1ac5deda7220)

**Reply by debuginn** [reply]
是这个意思 我不用来自定义 这个应该是模版的基础功能呈现 看看主题作者愿不愿意加
